template/data/build.gradle.kts

```
@@ -52,6 +45,16 @@ android {
         xmlReport = true
         xmlOutput = file("build/reports/lint/lint-result.xml")
     }
+
+    testOptions {
+        unitTests.all {
+            if (it.name != "testStagingDebugUnitTest") {
```

@ryan-conway Otherwise the tests for `:data` won't run 

```suggestion
            if (it.name != "testDebugUnitTest") {
```

Last remark, could you please add a PoW to the PR description? 🙏